lib/fetching/content-services/facebook-content-service.js

```diff
@@ -82,9 +85,21 @@ FacebookContentService.prototype._doFetch = function(options, callback) {
     return;
   }
 
+  var isTransient = function(error) {
+    if (error.code && _.contains(transientErrorCodes, error.code)) {
+      return true;
+    }
+    return false;
+  };
+
   this._doRequest(source, function(err, data) {
     if (err) {
-      self.emit('error', new Error(err.message));
+      var level = 'error';
+      if (isTransient(err)) level = 'warning';
+      process.nextTick(function() {
```

Comment:
Ah, alright. Seems fine.

Could always add the listener with `expectToEmitWarning` before causing the event.